app/code/Magento/Wishlist/Block/Customer/Wishlist/Item/Column/Cart.php

```diff
@@ -14,6 +20,31 @@
  */
 class Cart extends \Magento\Wishlist\Block\Customer\Wishlist\Item\Column
 {
+    /**
+     * @var View
+     */
+    private $productView;
+
+    /**
+     * @param \Magento\Catalog\Block\Product\Context $context
+     * @param \Magento\Framework\App\Http\Context $httpContext
+     * @param array $data
+     * @param ConfigInterface|null $config
+     * @param UrlBuilder|null $urlBuilder
+     * @param View|null $productView
+     */
+    public function __construct(
+        \Magento\Catalog\Block\Product\Context $context,
+        \Magento\Framework\App\Http\Context $httpContext,
+        array $data = [],
+        ?ConfigInterface $config = null,
+        ?UrlBuilder $urlBuilder = null,
+        ?View $productView = null
```

Comment:
@sivaschenko @torhoehn injected variables must not be nullable.

Reply:
@orlangur why not?
There are only two required parameters in `\Magento\Wishlist\Block\AbstractBlock::__construct`.
So theoretically it should also be possible to instantiate the `\Magento\Wishlist\Block\Customer\Wishlist\Item\Column\Cart` with only two params.
Thus we also need to make the corresponding parameters nullable in this case, don't we?

Reply:
@dmytro-ch because we never want to pass `null` explicitly. It is needed only as a default value.

Reply:
I get it now. It does make sense.
Thanks!